.gitmodules

```diff
@@ -9,7 +9,7 @@
 	url = https://github.com/tonymillion/Reachability.git
 [submodule "spec/cocoapods-integration-specs"]
 	path = spec/cocoapods-integration-specs
-	url = https://github.com/CocoaPods/cocoapods-integration-specs.git
+	url = https://github.com/benasher44/cocoapods-integration-specs.git
```

Comment:
I should have called this out more specifically, but that's what I meant about waiting on CocoaPods/cocoapods-integration-specs#76. I wait on that to be merged, and then I update the submodule references/pointers here right?